### PR TITLE
Add GA MP post

### PR DESCRIPTION
**Pull request by njam**
@fauvel please review

### CODE REVIEW

File: _posts/2015-02-18-googleanalytics-user-tracking.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Tracking User Events in Google Analytics from the Server-side"
```
Comment (fauvel) [inline _posts/2015-02-18-googleanalytics-user-tracking.markdown]: Without being too pedantic... I think you should rather say `Server Side` here (the dash is only used to build the adjective.)

File: _posts/2015-02-18-googleanalytics-user-tracking.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+Google Analytics' new *Universal Analytics* tracks [User IDs](https://developers.google.com/analytics/devguides/collection/analyticsjs/user-id) and therefore allows to connect multiple sessions (even across devices) from the same user.
+
+![GA Users](/img/posts/2015/ga-users.png)
+
+Additionally it's now possible to collect data over GA's new HTTP API [*Measurement Protocol*](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
+One can send POST requests to a URL to track *page views*, *events*, *e-commerce* etc.
+
+This allows one to track events for users that happen independently of any user interaction!
```
Comment (fauvel) [inline _posts/2015-02-18-googleanalytics-user-tracking.markdown]: I would say: This allows *you*.

File: _posts/2015-02-18-googleanalytics-user-tracking.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Imagine you bill your customers monthly for using your application. In the instant when the payment happens the user might not be using the application.
+Nevertheless you can send a POST request to Google Analytics to collect an e-commerce transaction.
+
+Universal Analytics knows the two parameters `cid` (*clientId*) to identify a session by its cookie, and `uid` (*userId*) to identify a user by your application's user-id.
+The `cid` is a required parameter, and `uid` is optional.
+
+How does GA handle such server-side sent hits? To find out the details I went ahead and sent some *events* to the REST API:
+
+- Example6: This event sent a `uid` and a `cid` that were both also used to browse the site with normal `analytics.js`.
+- Example7: This event sent a `uid` that was used with browsing the site, but a `cid` that was *made up*.
```
Comment (fauvel) [inline _posts/2015-02-18-googleanalytics-user-tracking.markdown]: I would say "used *while* browsing the site". And I think "a fake `cid`" would be more easily understandable?

**Comment by njam** [comment]
@fauvel thanks, adjusted!

**Pull request merged**